src/routes/[deviceUID]/dashboard/+page.svelte

```diff
@@ -43,6 +43,7 @@
     pm10_0: {}
   };
   let displayedReadings: AirnoteReading[];
+  let hideMap = 0;
```

Comment:
change this variable to a boolean so it's easier to understand whether the map is supposed to be shown or hidden in this file